recipes/pytest-tldr/meta.yaml

```
@@ -0,0 +1,34 @@
+{% set name = "pytest-tldr" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 008b7d53cabbce9d49ee7a92754ed4adafc056bc49ae01b257c2ffb1f5ce2408
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - pytest-runner
+  run:
+    - python
+    - pytest-runner
+
```

Please add a test section here as well
```suggestion
test:
  imports:
    - pytest_tldr
  commands:
    - pip check
  requires:
    - pip
```